sebs/openwhisk/openwhisk.py

```diff
@@ -168,7 +168,7 @@ def build_base_image(
         )
 
         for fn in os.listdir(directory):
-            if fn not in ("index.js", "__main__.py", "Main.java"):
+            if fn not in ("index.js", "__main__.py"):
```

Comment:
Why is the .java fil deleted now?

Reply:
Because Main.java is now in the same directory as the other files. They are all part of a Maven project and have been packaged into a fat jar for deployment on OpenWhisk. For functions larger than 48 MB, I think I will need to add Main.java again.